audiblelight/space.py

```diff
@@ -39,7 +39,7 @@ def load_mesh(mesh: Union[str, Path, trimesh.Trimesh]) -> trimesh.Trimesh:
         # elif not mesh.endswith(".glb"):
         #     logger.warning(f"Mesh file {mesh} is not a .glb file!")
         # Load mesh from file
-        loaded_mesh = trimesh.load_mesh(mesh, file_type=mesh.split(".")[-1])
+        loaded_mesh = trimesh.load_mesh(mesh, file_type=mesh.split(".")[-1], metadata={"fpath": mesh})
```

Comment:
`mesh.split(".")[-1]` assumes there’s always an extension and no dots in folders. Prefer `Path(mesh).suffix.`